addons/fcs/functions/fnc_keyUp.sqf

```diff
@@ -82,6 +83,11 @@ if (time - GVAR(time) > 1 and GVAR(time) != -1 and count _this < 3) then {
     _timeToLive     = getNumber (configFile >> "CfgAmmo" >> _ammoType >> "timeToLive");
     _simulationStep = getNumber (configFile >> "CfgAmmo" >> _ammoType >> "simulationStep");
 
+    _initSpeedCoef = getNumber(configFile >> "CfgWeapons" >> _weapon >> "initSpeed");
+    if (_initSpeedCoef < 0) then {
+        _initSpeed = _initSpeed * -_initSpeedCoef;
+    };
```

Comment:
> The original code did not support this either

I know, that was written before it was a thing ;)

I'd say yes, it's not a good idea on the part of the mod author, but if the game accepts it for ballistic, so should we.